### PR TITLE
Fix Helpers.extend

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka or @zazabe please review

**Comment by vogdb** [comment]
@zazabe please review

File: src/helpers.js [hunk 1]
```diff
@@ -6,17 +6,18 @@ var Helpers = {
    * @return {Object}
    */
   extend: function(destination, source) {
-    var sources = Array.prototype.slice.call(arguments, 1) || [];
+    if (source) {
```
Comment (zazabe) [inline src/helpers.js]: seems that the previous version was fine too, i mean, if there's no `source` parameter, `sources = []`, so nothing to loop on.

Comment (vogdb) [inline src/helpers.js]: It is for the case `Helpers.extend({}, null);`

Comment (zazabe) [inline src/helpers.js]: OK, so the docu should be updated?

Comment (vogdb) [inline src/helpers.js]: Do you mean to add `null`?
```js
  /**
   * @param {...Object|null} source
   */
```

Comment (zazabe) [inline src/helpers.js]: Yes, if it's expected...

**Comment by vogdb** [comment]
@zazabe please review

**Comment by zazabe** [comment]
lgtm

**Pull request merged**